### PR TITLE
Add optional presubmit to test cluster-sync against perf cluster

### PR DESCRIPTION
**Pull request by brianmcarey**
/cc @xpivarc @dhiller 
Signed-off-by: Brian Carey <bcarey@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -2358,3 +2358,80 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
```
Comment (xpivarc) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:2361]: Don't we need `optional: true`?

Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:2361]: IIRC if `always_run: true` is set, this job is conditional anyway, so it is implicitly optional. Can't find the doc link for this right now, though.

Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:2361]: There it is: https://docs.prow.k8s.io/docs/legacy-snapshot/prow/jobs/#requiring-jobs-for-auto-merge-through-tide

So in short: if neither `run_if_changed` nor `skip_if_only_changed` are set this job is already conditional and doesn't match any set, so it's optional.

**Review by dhiller — APPROVED** [review]
/approve

**Pull request merged**